### DISCUSSION TITLE
PEDIDO DA VOLTA DA FUNÇÃO "DIARIA" NO AZURACAST

### DISCUSSION THREAD

**Discussion by rricardodj** [AzuraCast/AzuraCast#5345] [Ideas]
Gostaria que a função diariamente que tinha no Azuracast volte, já que é de grande ajuda para programar spot comercial em horários específicos e dias da semana.
![IMG_20220504_084044358](https://user-images.githubusercontent.com/47946964/166856482-c4e6d347-7e71-4125-b58c-113b0263b22c.jpg)
Espero que possa ser atendido, para continuar a melhorar o azuracast.

**Comment by LordFader** [comment]
Olá Ricardo.
Obrigado pela participação.

Para facilitar e obteres maior ajuda da comunidade é conveniente utilizares a lingua Inglesa.
A comunidade é multi-cultural e fala-se muitas linguas por aqui, assim sendo o Inglês é a linguagem padrão para que todos entendam.
Tal como tu sou um mero utilizador e não faço parte da equipa de desenvolvimento.

Eu estou a utilizar a versão "Rolling Release #[17eb821]".
Penso que a função a que te referes mantem-se presente, no entanto está devidida pelos separadores.

![image](https://user-images.githubusercontent.com/4002168/166932518-8091e307-621a-4b96-a314-884b284128a1.png)
![image](https://user-images.githubusercontent.com/4002168/166932869-25052d07-9315-420a-afcf-3a92ec38fb51.png)
![image](https://user-images.githubusercontent.com/4002168/166933168-da3a97e8-a60c-42c9-8e2c-393310178965.png)

Espero ter ajudado... ou então causar ainda maior confusão.
:-D

Um abraço!|
